overlord/servicestate/service_control.go

```diff
@@ -142,8 +143,24 @@ func (m *ServiceManager) doServiceControl(t *state.Task, _ *tomb.Tomb) error {
 		flags := &wrappers.StartServicesFlags{
 			Enable: enable,
 		}
+		disabledServices := []string(nil)
```

Comment:
nitpick, but I think this is usually more idiomatic as 

```suggestion
		disabledServices := []string{}
```

Reply:
Done